src/Parser/Lexer.mll

```diff
@@ -11,6 +11,8 @@
   let _ =
     List.iter (fun (kwd, tok) -> Hashtbl.add keyword_table kwd tok)
       [ "split_ineq", RSPLIT_INEQ
+      ; "help", HELP
```

Comment:
We should make this help([a-Z A-Z]* as string) and then put a map from string to text somewhere.
This would be easier to maintain than adapting Parser all the time